newswires/app/controllers/ViteController.scala

```diff
@@ -61,12 +64,16 @@ class ViteController(
         .replaceAll("@csrf\\.value", csrf.value)
     }
 
-    def injectClientConfig(body: String): String = {
+    def injectClientConfig[A](
+        body: String
+    )(implicit request: UserRequest[A]): String = {
       val config =
         views.html.fragments.clientConfig(
           ClientConfig(
             FeatureSwitchProvider.clientSideSwitchStates,
-            stage = configuration.get[String]("stage")
+            stage = configuration.get[String]("stage"),
+            sendTelemetryAsDev =
+              request.user.email.startsWith("pete.faulconbridge@")
```

Comment:
hard-coded emails isn't ideal.. is it worth creating an SSM param or something to store emails?